server/models/users.js

```diff
@@ -27,7 +27,6 @@ const User = new Schema(
     nonce: {
       type: Number,
       default: () => Math.floor(Math.random() * 1000000),
-      select: false,
```

Comment:
@sagara11 cái này sao lại bỏ đi nhỉ, mình có cần phải truy vấn cả nonce đâu

Reply:
@SalamanderHP có nhẽ chỗ hiện ra i am signing this nonce